conda/recipes/dask-cuda/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
     - setuptools
   run:
     - python
-    - dask >=2.22.0
-    - distributed >=2.22.0
+    - dask >=2.22.0,<=2021.09.1
+    - distributed >=2.22.0,<=2021.09.1
```

Comment:
I think we should pin to _exactly_ `=2021.09.1`. There have been changes that are breaking in older versions, e.g. https://github.com/rapidsai/dask-cuda/pull/722 .

Reply:
Any objections @jakirkham @quasiben to my comment above?

Reply:
Am happy to defer to you here Peter :)

Reply:
Let's then pin to `=2021.09.1`, I don't see any compelling reason to support half-functional versions.

Could you do that @galipremsagar ? And before I forget, we also need to set https://github.com/rapidsai/dask-cuda/blob/28ce42701fdac4bb1f06ca7705885e8f1f657c0b/ci/gpu/build.sh#L36 to `0`. Same with other repos.

Reply:
```suggestion
    - dask=2021.09.1
    - distributed=2021.09.1
```